src/invidious/channels/community.cr

```diff
@@ -185,10 +188,40 @@ def fetch_channel_community(ucid, continuation, locale, format, thin_mode)
                           end
                         end
                       end
-                      # TODO
-                      # when .has_key?("pollRenderer")
-                      #   attachment = attachment["pollRenderer"]
-                      #   json.field "type", "poll"
+                    when .has_key?("pollRenderer")
+                      attachment = attachment["pollRenderer"]
+                      json.field "type", "poll"
+                      json.field "totalVotes", short_text_to_number(attachment["totalVotes"]["simpleText"].as_s.split(" ")[0])
+                      json.field "choices" do
+                        json.array do
+                          attachment["choices"].as_a.each do |choice|
+                            json.object do
+                              json.field "text", choice["text"]["runs"][0]["text"].as_s
+                              # A choice can have an image associated with it.
+                              # Ex post: https://www.youtube.com/post/UgkxD4XavXUD4NQiddJXXdohbwOwcVqrH9Re
+                              if choice["image"]?
+                                thumbnail = choice["image"]["thumbnails"][0].as_h
+                                width = thumbnail["width"].as_i
+                                height = thumbnail["height"].as_i
+                                aspect_ratio = (width.to_f / height.to_f)
+                                url = thumbnail["url"].as_s.gsub(/=w\d+-h\d+(-p)?(-nd)?(-df)?(-rwa)?/, "=s640")
+                                qualities = {320, 560, 640, 1280, 2000}
```

Comment:
This array is present multiple times across the file. Making a file level constant is recommended:
```crystal
private IMAGE_QUALITIES = {320, 560, 640, 1280, 2000}
```